cmd/proxy/config/swagger/openapi.json

```diff
@@ -0,0 +1,3016 @@
+        ],
+        "summary": "get a block from a specified shard and with a specified hash",
+        "parameters": [
+          {
+            "name": "shard",
+            "in": "path",
+            "description": "the shard ID to look after",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "hash",
+            "in": "path",
+            "description": "the hash to look after",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/block/by-round/{round}": {
+      "get": {
+        "tags": [
+          "block"
+        ],
+        "summary": "returns all blocks by round.",
+        "parameters": [
+          {
+            "name": "round",
+            "in": "path",
+            "description": "the round to look after",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/block/{shard}/by-nonce/{nonce}": {
+      "get": {
+        "tags": [
+          "block"
+        ],
+        "summary": "return a block by nonce and shard.",
+        "parameters": [
+          {
+            "name": "shard",
+            "in": "path",
+            "description": "the shard ID to look after",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "nonce",
+            "in": "path",
+            "description": "the hash to look after",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/block/{shard}/by-nonce/{nonce}?withTxs=true": {
+      "get": {
+        "tags": [
+          "block"
+        ],
+        "summary": "returns a block by nonce, with transactions included.",
+        "parameters": [
```

Comment:
here we need also a parameter for `shard`

Reply:
added